api/controllers/v1/cave/find-all.js

```diff
@@ -1,46 +1,37 @@
-const CaverService = require('../../../services/CaverService');
 const ControllerService = require('../../../services/ControllerService');
-const { toCave } = require('../../../services/mapping/converters');
+const { toCaveShort } = require('../../../services/mapping/converters');
 const { toListFromController } = require('../../../services/mapping/utils');
 const NameService = require('../../../services/NameService');
 
 module.exports = async (req, res) => {
-  const parameters = {};
+  const params = {
+    controllerMethod: 'CaveController.findAll',
+    searchedItem: 'all caves',
+    notFoundMessage: 'No caves found.',
+  };
+  try {
+    const parameters = {
+      // TODO Add parameters ?
```

Comment:
ControllerService params are not very useful, I think: one day the `treat()` and `treatAndConvert()` methods will be totally reworked. They were supposed (I guess) to handle a lot of default cases (like errors or not found), but in practice, it's better to handle such cases in the controller. 

Anyway, add at least `searchedItem` and `controllerMethod` for debug purpose.

Reply:
Agree, the Controller Service should be reworked
Here the variable parameters is used in the TCave.find() not for the ControllerService.

Reply:
Oh sorry, I misread.

I think the front doesn't use this controller / route. Maybe we should add a default limit to avoid users querying too much the route ? Like 100 caves ?